pkgs/top-level/all-packages.nix

```diff
@@ -27082,13 +27082,13 @@ with pkgs;
 
     # C++20 is required, darwin has Clang 7 by default, aarch64 has gcc 9 by default
     stdenv = if stdenv.isDarwin
-      then llvmPackages_12.libcxxStdenv
+      then clang12Stdenv
```

Comment:
That is an alias https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/aliases.nix#L171=

It should be `llvmPackages_12.stdenv`. Also changing the darwin stdenv should be done with care and not rushed in a seemingly unrelated package.

Reply:
It's the same env, just is looks nicer with gcc10Stdenv